MOxUnit/moxunit_runtests.m

```diff
@@ -126,6 +130,18 @@
             mocov_params{k*2}=params.(key);
         end
 
+        % Now do all the keys which map to cell arrays. These are for
+        % arguments which occur multiple times. For these, we make a
+        % parameter pair input for every element in the cell.
+        keys=all_keys(msk_cover & msk_iscell);
+        for k=1:numel(keys)
+            key=keys{k};
+            for c=1:numel(params.(key))
+                mocov_params{end+1}=['-' key];
+                mocov_params{end+1}=params.(key){c};
+            end
+        end
```

Comment:
This does not seem optimal:
- this introduces multiple repeats of the ``params.(key)`` expression.
- mocov_params grows in the loop.
- some code duplication with setting mocov_params.

how about the following:

```
    keys=all_keys(msk_cover);
    n_keys=numel(keys);
    mocov_params_cell=cell(1,n_keys);
    for k=1:n_keys
        key=keys{k};
        value=params.(key);

        dash_key=['-' key];
        if ischar(value)
            param_elem={dash_key,value};
        elseif iscell(value)
            n_values=numel(value);
            param_elem_matrix=[repmat({dash_key},1,n_values);...
                                       value(:)'];
            param_elem=param_elem_matrix(:)';
        else
            assert(false,'expected string or cell input');
        end

        mocov_params_cell{k}=param_elem;
    end

    mocov_params=cat(2,mocov_params_cell{:});
```